pages/alerts.rst

```diff
@@ -284,11 +284,57 @@ Client URL
 .. image:: /images/alerts_pagerduty_notification.png
 
 
+.. _alert_notification_Slack Notification:
+
+Slack alert notification
+------------------------
+The Slack alert notification allows you to send notifications using your `Webhook URL <https://api.slack.com/messaging/webhooks>`__ to your slack workspace
```

Comment:
I think the URL link would make more sense below in the configuration option for the Webhook URL.  In fact, this might read better as "The Slack alert notification allows you to send notifications to your slack workspace in response to events in your Graylog server."

Reply:
https://github.com/Graylog2/documentation/pull/986